TDS_3/include/CGAL/Triangulation_data_structure_3.h

```diff
@@ -936,24 +936,40 @@ class Triangulation_data_structure_3
     }
 
     void operator()(Cell_handle c) {
+      std::cout <<  " [] Visiting cell: " << &*c << std::endl;
```

Comment:
I suggest surrounding all use of `std::cout` by
```C++
#ifdef CGAL_DEBUG_TDS_3_<SOMETHING>
  std::cout << "bla";
#endif //  CGAL_DEBUG_TDS_3_<SOMETHING>
```

That way you can actually keep the debugging code in CGAL, for a later use.

Note also that in my opinion `std::cerr` or `std::clog` should be preferred instead of `std::cout`.

Reply:
This commit was just to run it on Andreas' machine and I'll reset hard after.